integration_test/test_videoroom/test/integration/stampede_test.exs

```diff
@@ -1,5 +1,5 @@
 defmodule TestVideoroom.Integration.ClientTest do
```

Comment:
`ClientTest` ->` BasicTest`? Or something better but I don't have other ideas at the moment